bindings/python/pyiree/rt/vm_test.py

```diff
@@ -126,6 +126,7 @@ def test_add_scalar(self):
     logging.info("abi: %s", abi)
 
     inputs = abi.raw_pack_inputs((5, 6))
+    logging.info("serialize_inputs: %s", abi.serialize_vm_list(inputs))
```

Comment:
These aren't huge log statements because the inputs have at most four elements. It's indirectly testing that you can pass the output of `abi.raw_pack_inputs` to `abi.serialize_vm_list` and also serves to document that capability via the test. (For me a lot of figuring out how the bindings work was just looking at the tests).

Reply:
Lol whoops I missed that these are just in a test. If you want to test that you can pass the output of one function to another, I recommend having a test for that specifically though :-)